### ISSUE TITLE
Sending a form in an array to a view wipes the other messages

### ISSUE THREAD

**Issue by Azenet** [FriendsOfSymfony/FOSRestBundle#1093]
I try to show errors in a form along with a status message so my code looks like this:
```
return $this->handleView($this->view(array('error' => 'missing_data', 'form' => $form)));
```

I expect to have ```{"error":"missing_data","form":{"children":....}}``` but I just get ```{"children":....}```.

Tried with dev-master FOSRestBundle at time of writing (FOSRestBundle@2d9bf77fdb4789e8f76c4fef072bd5b862f3b882)

**Comment by lsmith77** [comment]
yeah this is expected .. we didn't consider the use case of wanting to customize the error message this way ..
